api/src/pcapi/core/fraud/api.py

```diff
@@ -486,7 +487,7 @@ def _handle_duplicate(fraud_items: list[models.FraudItem], fraud_check: models.B
         actor=None,
         comment="Compte automatiquement suspendu pour suspicion de doublon",
     )
-    _send_duplicate_fraud_detection_mail(user, duplicate_beneficiary)
+    fraud_emails.send_duplicate_fraud_detection_mail(user, duplicate_beneficiary)
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Du coup il faut supprimer `_send_duplicate_fraud_detection_mail` de ce fichier